packages/core/test/scp-cf/client-credentials-token-cache.spec.ts

```diff
@@ -4,18 +4,18 @@ import { clientCredentialsTokenCache } from '../../src/scp-cf';
 describe('ClientCredentialsTokenCache', () => {
   it('should return token when valid, return undefined otherwise', () => {
     const clock = install();
-    const now = Date.now();
     const validToken = {
       access_token: '1234567890',
       token_type: 'UserToken',
-      expires_in: now + 100000,
+      // x second later
```

Comment:
I do not think that this comment belongs here, because it is documentation of the jwt. If you think this information should be added somewhere, maybe the knowledge-base is a better place

Reply:
moved to the js doc of the interface